tasks/dom_massager.js

```diff
@@ -20,8 +20,9 @@ module.exports = function(grunt) {
 			writeDom: false,
 			xmlMode: false,
 			normalizeWhitespace: false,
-			selectors:{},
-			cheerioHook:null
+			decodeEntities: true,
```

Comment:
wont this be changing the existing behavior for people that were not anticipating this.

Reply:
Sorry, my bad.  Yes, your default was true.

Reply:
pretty sure its false: https://github.com/fb55/htmlparser2/wiki/Parser-options

Reply:
Cheerio defaults this to true: https://github.com/cheeriojs/cheerio

Reply:
got it. specifically https://github.com/cheeriojs/cheerio/blob/master/lib/cheerio.js#L109

which is super annoying cause their docs explicitly say they default to what Parser-options uses.